spras/config/util.py

```diff
@@ -0,0 +1,19 @@
+from enum import Enum
+from typing import Any
+
+
+# https://stackoverflow.com/a/76883868/7589775
+class CaseInsensitiveEnum(str, Enum):
+    """
+    We prefer this over Enum to make sure the config parsing
```

Comment:
I'm a little confused by this comment. I thought you explicitly have to define an Enum class when you want to use Enums? Is there a way to use Enum by itself?

Reply:
I don't think I'm properly understanding this review comment, but you can extend Enums before using them.